acceptance/Rakefile

```diff
@@ -233,16 +233,19 @@ def sha
   ENV['SHA']
 end
 
+# Return the host config file for beaker to use
+# If TEST_TARGET is set, pass it to beaker-hostgenerator to generate a file
+# else, assume CONFIG is set to the path of an existing host config file
 def config
-  if (ENV['TEST_TARGET']) && (not ENV['CONFIG'])
+  if (ENV['TEST_TARGET'])
```

Comment:
ok.  so it turns out these jobs are sending both CONFIG and TEST_TARGET.
this should probably be the same as facter's current work around:
```
def hosts
  if ENV['HOSTS']
    ENV['HOSTS']
  else
    puts 'Warning: environment variable CONFIG deprecated. Please use HOSTS to match beaker options.'
    ENV['CONFIG']
  end
end
[...]

 # The 'if ...' suffix conditional below is a temporary measure which should go
  # away once the pipeline changes are made so that TEST_TARGET is always
  # intended for use with beaker-hostgenerator. So please remove that
  # conditional (and this comment) once that pipeline change is made.
  target = ENV['TEST_TARGET'] if ENV['TEST_TARGET'] && ENV['TEST_TARGET'][-1,1] == 'a'
  if target
    cli = BeakerHostGenerator::CLI.new([target, '--disable-default-role', '--osinfo-version', '1'])
    ENV['HOSTS'] = "tmp/#{target}-#{SecureRandom.uuid}.yaml"
    FileUtils.mkdir_p('tmp')
    File.open(hosts, 'w') do |fh|
      fh.print(cli.execute)
    end
    config_opt = "--hosts=#{hosts}"
  elsif hosts
    config_opt = "--hosts=#{hosts}"
  end
```